scipy/special/_special_ufuncs.cpp

```diff
@@ -16,10 +17,45 @@
 
 using namespace std;
 
-// This is needed by sf_error, it is defined in the Cython "_ufuncs_extra_code_common.pxi" for "_generate_pyx.py".
-// It exists to "call PyUFunc_getfperr in a context where PyUFunc_API array is initialized", but here we are
-// already in such a context.
-extern "C" int wrap_PyUFunc_getfperr() { return PyUFunc_getfperr(); }
```

Comment:
Thoughts on splitting the gufuncs into a separate `_gufuncs.cpp`? I'd find it easier to understand these files at a glance if they were kept separate, and I think it would make `_special_ufuncs.cpp` less intimidating to beginners.

Reply:
Sure, happy to do that.